vendor/github.com/docker/docker/api/types/swarm/container.go

```diff
@@ -55,6 +55,7 @@ type ContainerSpec struct {
 	User            string                  `json:",omitempty"`
 	Groups          []string                `json:",omitempty"`
 	Privileges      *Privileges             `json:",omitempty"`
+	Init            bool                    `json:",omitempty"`
```

Comment:
As commented in https://github.com/moby/moby/issues/34529#issuecomment-330806345 - SwarmKit changes are merged in moby/moby, so can you open a pull request there to make the required changes in moby/moby?

Reply:
(you, or @denverdino ) 😇